.devcontainer/devcontainer.json

```diff
@@ -4,25 +4,29 @@
         "../development/docker-compose.yml",
         "../development/docker-compose.postgres.yml",
         "../development/docker-compose.dev.yml",
-        "../development/docker-compose.vscode-rdb.yml"
+        "../development/docker-compose.vscode-rdb.yml",
+        "../development/docker-compose.static-ports.yml"
     ],
     "features": {
-        "ghcr.io/devcontainers/features/docker-outside-of-docker:1": { }
+        "ghcr.io/devcontainers/features/docker-outside-of-docker:1": {}
     },
     "service": "nautobot",
     "shutdownAction": "stopCompose",
+    "containerEnv": {
+        "INVOKE_NAUTOBOT_PROJECT_NAME": "development"
+    },
     "customizations": {
         "vscode": {
             "extensions": [
                 "charliermarsh.ruff",
                 "DavidAnson.vscode-markdownlint",
-                "eamodio.gitlens",
```

Comment:
This is a paid extension and VSCode native git support has gotten significantly better since this was added